.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,8 +23,15 @@ Please also:
 - [ ] This PR meets the Acceptance Criteria in the JIRA story
 - [ ] The JIRA story has been moved to Dev Review
 - [ ] This PR includes appropriate unit tests
-- [ ] I have run and tested this change locally
-- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
-- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
-- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
+- [ ] I have added explanatory comments where the logic is not obvious
+- [ ] I have run and tested this change locally, and my testing process is described here
 - [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
+- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
+- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
+- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
+- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified/updated the appropriate API consumers
```

Comment:
All of the examples below are cases of "update", and none are "notify" - except **maybe** the RDR export in certain cases. 

Maybe just change this to "update", and then add a wiki link in the RDR bullet, which has details on communication in that case: https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export)

I just added that wiki as I realized we never checked it in.

Reply:
oh thanks!  I was looking for such a thing